packages/simulation-sdk/package.json

```diff
@@ -18,6 +18,8 @@
     "build": "tsc --noEmit && tsc --build tsconfig.build.json"
   },
   "dependencies": {
+    "@types/lodash.merge": "^4.6.9",
+    "lodash.merge": "^4.6.2",
```

Comment:
Why?

Reply:
Also types should only be dev

Reply:
I use it in `simulation-sdk` test fixtures so both can be in dev